thirdeye/thirdeye-frontend/app/styles/shared/_styles.scss

```diff
@@ -541,18 +541,30 @@ body {
       font-size: 13px;
     }
 
-    &--smaller {
-      font-size: 12px;
-      margin-left: -2px;
-    }
-
     &--shadow {
       position: absolute;
       color: app-shade(black, 2);
       margin-left: -18px;
     }
   }
 
+  &__list-item {
+    font-size: 12px;
+    margin-left: -2px;
+  }
+
+  &__value-label {
+    font-size: 13px;
+
+    &--up {
+      color: green;
```

Comment:
To keep a consistent UI, let's pick colors from here https://github.com/linkedin/pinot/blob/master/thirdeye/thirdeye-frontend/app/styles/wrapper/_color.scss

Reply:
will do.